sabnzbd/utils/ssdp.py

```diff
@@ -34,9 +41,45 @@ def __init__(self, host, server_name, url, description):
         self.__url = url
         self.__description = description
         self.__myhostname = socket.gethostname()
-        # uuid stays the same as long as hostname and ip address stay the same:
-        self.__uuidXML = uuid.uuid3(uuid.NAMESPACE_DNS, self.__myhostname + self.__host)
-        self.__uuidSSDP = uuid.uuid3(uuid.NAMESPACE_DNS, self.__host + self.__myhostname)
+        # a steady uuid: stays the same as long as hostname and ip address stay the same:
+        self.__uuid = uuid.uuid3(uuid.NAMESPACE_DNS, self.__myhostname + self.__host)
+
+        # descriptionxmlURL = self.__url + "/description.xml"
+
+        # Create the SSDP broadcast message
+        self.__mySSDPbroadcast = f"""NOTIFY * HTTP/1.1
+HOST: 239.255.255.250:1900
+CACHE-CONTROL: max-age=60
+LOCATION: {self.__url}/description.xml
+SERVER: SABnzbd
```

Comment:
Use `self.__server_name`

Reply:
It must be a URL. server_name is just "SABnzbd". So, IMHO, won't work

![image](https://user-images.githubusercontent.com/1273502/97086042-44447100-1621-11eb-981e-7c8e50a351c5.png)